packages/databases-collections/src/components/rename-collection-modal/rename-collection-modal.tsx

```diff
@@ -37,11 +38,36 @@ const progressContainerStyles = css({
 
 type ModalState = 'input-form' | 'confirmation-screen';
 
+function ConfirmationModalContent({
+  areSavedQueriesAndAggregationsImpacted,
+}: {
+  areSavedQueriesAndAggregationsImpacted: boolean;
+}) {
+  return (
+    <Banner variant="warning" data-testid="rename-collection-modal-warning">
+      Renaming collection will result in loss of any unsaved queries, filters or
+      aggregation pipeline.
+      {areSavedQueriesAndAggregationsImpacted && (
+        <Fragment>
+          <br />
+          <br />
+          <b>
+            {' '}
+            Additionally, any saved queries or aggregations targeting this
+            collection will need to be remapped to the new namespace.
+          </b>
+        </Fragment>
+      )}
+    </Banner>
```

Comment:
I'd like someone to look over how I built this UI. I'm not sure if I'm following best practices. 

One issue that I noticed is that we are currently using @leafygreen-ui/typography 1.4, but the styles I need are defined in 2.x.  So instead of styling double text with the leafygreen UI semibold style, I used the `b` tag here. Alternatively, we could upgrade to 2.X and use the provided style.